packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -218,6 +218,11 @@ class AnimationController extends Animation<double>
     notifyListeners();
     _checkStatusChanged();
   }
+  
+  /// Stops the animation and sets it back to its beginning
```

Comment:
Instead of "beginning", maybe "[lowerBound]"?
Also, please end the documentation with a period. :-)

Reply:
Counter point, lowerBound is a technical term, one that if better understood by the user, would counteract the need for the convenience method. Prioritizing readability and intuitability.

I could probably phrase it differently to be both technical and readable.

`/// Stops the animation and sets its value to the [lowerBound], resetting to the beginning.`

Something like that? I want to be careful to be correct, because lowerBound is a term related to the controller itself not the animation.

Reply:
That's a very valid point.

Feel free to make this a multi-paragraph explanation if that's what it takes. :-)